app/src/main/java/in/co/rajkumaar/amritarepo/activities/LaunchingActivity.java

```diff
@@ -624,7 +559,7 @@ protected void onResume() {
                 .addOnSuccessListener(appUpdateInfo -> {
                     // If the update is downloaded but not installed,
                     // notify the user to complete the update.
-                    if(appUpdateInfo.updatePriority()>3){
+                    if (appUpdateInfo.clientVersionStalenessDays() > DAYS_FOR_IMMEDIATE_UPDATE) {
```

Comment:
Add a condition to check if stale days is null.

Reply:
done